spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/Span.java

```diff
@@ -31,16 +31,28 @@
  * Class for gathering and reporting statistics about a block of execution.
  * <p/>
  * Spans should form a directed acyclic graph structure. It should be possible to keep
- * following the parents of a span until you arrive at a span with no parents.
+ * following the parents of a Span until you arrive at a Span with no parents.
  * <p/>
  *
+ * <p>Spans can be either annotated with tags or logs.</p>
+ *
+ * <p>An <b>Annotation</b> is used to record existence of an event in time. Below you can find some
+ * of the core annotations used to define the start and stop of a request:</p>
+ * <ul>
+ *     <li><b>cs</b> - Client Sent: {@link org.springframework.cloud.sleuth.event.ClientSentEvent}</li>
```

Comment:
prefer `{@link org.springframework.cloud.sleuth.event.ClientSentEvent Client Sent}` syntax? a bit shorter and nicer to look at imho